build/kubernetes/minikube_setup.sh

```diff
@@ -1,49 +1,40 @@
 #!/bin/bash
 set -v
 
-ci_bin=$GOPATH/src/github.com/coredns/ci/build
+# Setup Kubectl
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
 
-# Start local docker image repository
+# Setup Minikube
+mkdir -p ${HOME}/.kube
+touch ${HOME}/.kube/config
+curl -Lo minikube https://github.com/kubernetes/minikube/releases/download/${MINIKUBE_VERSION}/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+
+# Start a local docker repository
 docker run -d -p 5000:5000 --restart=always --name registry registry:2.6.2
```

Comment:
We don't need to do this since we test on a single node (i didn't realize this when I did this 2? years ago) ... it would save time (and code) to skip this step and load images directly from local docker instance.  I suppose we can clean this up later.